nixos/modules/services/torrent/transmission.nix

```diff
@@ -310,8 +313,11 @@ in
       after = [ "network.target" ] ++ optional apparmor.enable "apparmor.service";
       requires = optional apparmor.enable "apparmor.service";
       wantedBy = [ "multi-user.target" ];
-      environment.CURL_CA_BUNDLE = etc."ssl/certs/ca-certificates.crt".source;
-      environment.TRANSMISSION_WEB_HOME = lib.mkIf (cfg.webHome != null) cfg.webHome;
+
+      environment = {
+        CURL_CA_BUNDLE = etc."ssl/certs/ca-certificates.crt".source;
```

Comment:
Should we reference the `cacert` package directly instead?

Reply:
I mean, I don't think there's any guarantee that there'll be anything at `/etc/ssl/certs/ca-certificates.crt`, though, right? Not unless *something* installs `cacert`.